### PR TITLE
Add Dialect notifications

### PR DESCRIPTION
**Pull request by dshuvalov**
# Feature
- update dialect's packages and legacy modal notifications
- integrate Dialect notifications into hub components

## Dialect's notifications modal in legacy UI

https://user-images.githubusercontent.com/16071062/198609649-8c63765a-1cab-43fa-9176-df5027ad5939.mov

## Dialect's notifications modal in the hub components
Снимок экрана 2022-10-28 в 16 15 38
Снимок экрана 2022-10-28 в 16 15 53
Снимок экрана 2022-10-28 в 16 16 00
Снимок экрана 2022-10-28 в 16 15 32

### CODE REVIEW

**Comment by dshuvalov** [comment]
How it looks after pulling main branch
Снимок экрана 2022-10-28 в 17 53 20
Снимок экрана 2022-10-28 в 17 55 40

**Review by nramadas — CHANGES_REQUESTED** [review]
Looks great! Just one small change requested. Thanks!

File: hub/providers/WalletSelector/index.tsx [hunk 1]
```diff
@@ -249,7 +249,7 @@ export function WalletSelector(props: Props) {
 
   return (
     <ConnectionProvider endpoint={cluster.endpoint}>
-      <_WalletProvider wallets={supportedWallets}>
+      <_WalletProvider wallets={supportedWallets} autoConnect>
```
[the remark this answers is not in the document]
Comment (dshuvalov) [reply hub/providers/WalletSelector/index.tsx:252]: No problem. I can explain. We use WalletProvider and useWallet hook from the "@solana/wallet-adapter-react" for configuring DialectSolanaSdk from our package, we use auto connect for retrieving wallet info if a user had already signed in to Solana wallet (phantom app for example). Without this preference, user has to sign in to wallet every time

File: hub/components/GlobalHeader/User/DialectNotifications/Notifications.constants.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import {
+  defaultVariables,
+  IncomingThemeVariables,
+} from '@dialectlabs/react-ui';
+import * as anchor from '@project-serum/anchor';
```
[the remark this answers is not in the document]
Comment (dshuvalov) [reply hub/components/GlobalHeader/User/DialectNotifications/Notifications.constants.ts]: Fixed. Thanks!

**Review by nramadas — APPROVED** [review]

**Pull request merged**